doc/src/site/sphinx/migration_guide.rst

```diff
@@ -47,6 +47,19 @@ From 3.30 to 3.32
   than 0, validation metrics if validation frame has been specified ( splitRatio was set and lower than 1 ) and nfolds was 0
   and training metrics otherwise ( splitRatio is 1 and nfolds is 0).
 
+- The methods ``join``, ``innerJoin``, ``outerJoin``, ``leftJoin`` and ``rightJoin`` in ``water.support.JoinSupport``
+  are removed together with their encapsulating class. The enum ``water.support.munging.JoinMethod`` is also removed.
+  In order to perform joins, please use the following methods:
+
+  - Inner join: ``ai.h2o.sparkling.H2OFrame(idOfLeftFrame).innerJoin(rightFrame, method)``
+  - Outer join: ``ai.h2o.sparkling.H2OFrame(idOfLeftFrame).outerJoin(rightFrame, method)``
+  - Left join: ``ai.h2o.sparkling.H2OFrame(idOfLeftFrame).leftJoin(rightFrame, method)``
+  - Right join: ``ai.h2o.sparkling.H2OFrame(idOfLeftFrame).rightJoin(rightFrame, method)``
+
+
+The ``JoinMethod`` enum is no longer needed, but string is expected instead in ``method`` argument.
```

Comment:
nit: `method` argument -> the `method` argument

Reply:
👍